src/main/java/dev/tycho/stonks/managers/DatabaseHelper.java

```diff
@@ -49,23 +51,26 @@ public DatabaseHelper(Stonks plugin, DatabaseManager databaseManager) {
   }
 
   public void createCompany(Player player, String companyName) {
+    //Prevent the player from spamming companies
     if (!player.isOp() && playerCompanyCooldown.containsKey(player.getUniqueId()) && (System.currentTimeMillis() - playerCompanyCooldown.get(player.getUniqueId())) < COMPANY_CREATION_COOLDOWN) {
-      sendMessage(player, "You cannot make a company for another " + Util.convertString(COMPANY_CREATION_COOLDOWN - (System.currentTimeMillis() - playerCompanyCooldown.get(player.getUniqueId()))));
+      player.sendMessage(ChatColor.RED + "You cannot make a company for another " +
```

Comment:
Revert this

Reply:
https://github.com/schlatt-co/stonks/blob/a34c87a8a16fdce21ba0462044aedb07a35af01b/src/main/java/dev/tycho/stonks/managers/DatabaseHelper.java#L56